vendor/bat-native-ads/src/bat/ads/internal/base/database/database_bind_util.cc

```diff
@@ -28,7 +28,7 @@ std::string BuildBindingParameterPlaceholders(const size_t parameters_count,
                                               const size_t values_count) {
   DCHECK_NE(0UL, values_count);
 
-  const std::string value = BuildBindingParameterPlaceholder(parameters_count);
+  std::string value = BuildBindingParameterPlaceholder(parameters_count);
```

Comment:
why remove const here?

Reply:
const-ness of value prevented an automatic move

Reply:
move semantics as above

Reply:
In this case the compiler will either use NVRO (not guaranteed) and if need be fall back to move

Reply:
with the const, we get a copy